src/tests/Microsoft.PowerFx.Interpreter.Tests/PlainTextTests.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.PowerFx.Core.Tests;
+using Xunit;
+
+namespace Microsoft.PowerFx.Tests
+{ 
+    public class PlainTextTests : PowerFxTest
+    {
+        [Theory]
+        [InlineData("PlainText(\"Test<br id = 'br1'>break line</  BR>\")", "Test\r\nbreak line")]
+        [InlineData("PlainText(\"1<br>2<li>3\")", "1\r\n2\r\n3")]
+        [InlineData("PlainText(\"1<BR/>2<li/>3\")", "1\r\n2\r\n3")]
+        [InlineData("PlainText(\"Test<p >double</  p><div >break line</ Div>\")", "Test\r\n\r\ndouble\r\n\r\nbreak line")]
+        [InlineData("PlainText(\"1<div>2<p>3<tr>4\")", "1\r\n\r\n2\r\n\r\n3\r\n\r\n4")]
+        [InlineData("PlainText(\"1<Div/>2<p/>3<Tr/>4\")", "1\r\n\r\n2\r\n\r\n3\r\n\r\n4")]
+        [InlineData("PlainText(\"1<  br  >2<  Li />3\")", "1\r\n2\r\n3")]
+        [InlineData("PlainText(\"1<div  class = 'div2'>2< P />3<  tr>4\")", "1\r\n\r\n2\r\n\r\n3\r\n\r\n4")]
+        public async Task PlainTextFunctionTest(string inputExp, string expectedResult)
```

Comment:
Can these be merged into PlainText.txt?  EG, we may not need any dedicated C#  test.

Reply:
I don't think the .txt tests support \r\n in the output. We may be able to work around it by using something like

    >> Substitute(Substitute(PlainText(\"1<  br  >2<  Li />3\")", "1\r\n2\r\n3"), Char(13), "\r"), Char(10), "\n")
    "1\r\n2\r\n3"

Or we may want to see if we can "fix" the test runner to account for multi-line strings

Reply:
Thanks for suggestion. Removed this file and add to test txt file.